src/Extensions/Diagnostics.HealthChecks/Internal/RestHealthCheckPublisher.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Omex.Extensions.Abstractions.ExecutionContext;
+using Microsoft.ServiceFabric.Client;
+using Microsoft.ServiceFabric.Client.Http;
+using sfc = Microsoft.ServiceFabric.Common;
+using sfh = System.Fabric.Health;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	/// <summary>
+	/// Just mock
+	/// </summary>
+	public class RestHealthCheckPublisher : HealthCheckPublisher
```

Comment:
I would suggest making class `internal` and just provide public method to register it on ServiceCollection. Is this way you would reduce possibility of the wrong usage of the type and remove the need of comments :)